data/unitLabel4Text.xml

```diff
@@ -9955,7 +9955,7 @@ period a year ago. Revenue was nearly unchanged at $20.40bn, compared with $20.4
 <h>CO2 emissions from gaseous fuel consumption (% of total) in France was 24.20 as of 2009.</h>
 <ht>
 CO2 emissions from gaseous fuel consumption (% of total) CO2 emissions from gaseous fuel consumption (% of total) in France was 24.20 as of 2009. Its highest value</ht>
-<u>
+<u>kiloton</u>
```

Comment:
This is incorrect.  The unit here is not kiloton, but percent.

Reply:
Yes, thanks. I have updated it.
Please let me know if you see any other issue, else I will create a new pull request.